db/migrate/20220726111449_drop_classification_policies_table.rb

```diff
@@ -0,0 +1,5 @@
+class DropClassificationPoliciesTable < ActiveRecord::Migration[7.0]
```

Comment:
Would you mind updating the commit message to include the link? You did on the other one but this one got missed

Reply:
Looks like I forgot to copy it into the sentence 🤦‍♂️.  Have updated the message now.